Projects/CMakeLists.txt

```diff
@@ -134,12 +143,12 @@ set_property(TARGET ${LIBRARY_NAME} PROPERTY VERSION ${APP_VERSION})
 target_compile_definitions(${LIBRARY_NAME} PRIVATE EXTERNALMEDIA_FLUIDPROP=$<IF:$<BOOL:${FLUIDPROP}>,1,0>)
 target_compile_definitions(${LIBRARY_NAME} PRIVATE EXTERNALMEDIA_COOLPROP=$<IF:$<BOOL:${COOLPROP}>,1,0>)
 target_compile_definitions(${LIBRARY_NAME} PRIVATE EXTERNALMEDIA_MODELICA_ERRORS=1) # Use 0 for a shared library and 1 for a static library
-#target_compile_definitions(${LIBRARY_NAME} PRIVATE EXTERNALMEDIA_LIBRARY_EXPORTS) # Use this for a shared library
+

```

Comment:
> Thank you for your updates.
> 
> Could you please explain why you create an additional target *_DLL? For me this should be an option - either one wants the shared or the static library, but creating both probably only creates confusion.
> 
> I am going to include some of our suggestions in v3.3.0 and then we can focus on OpenModelica and CoolProp v6.4.2 in v3.4.0.

In the recent update in openmodelica,  the new-frontend (NF) just requires such a DLL to initialize any constant by an external function. So, I just took a conservative manner to preserve the original cmake script as much as possible, and to add up the DLL only in case of openmodelica; if you like the DLL for a major option of the build process, we can define such an option to generate the DLL instead of the static library. Do you prefer that manner?